unit-tests/py/rspy/unifi.py

```diff
@@ -14,14 +14,16 @@ def usage():
         print( 'Syntax: unifi [options]' )
         print( '        Control the Unifi Ubiquiti hub' )
         print( 'Options:' )
-        print( '        --enable       Enable all ports' )
-        print( '        --disable      Disable all ports' )
-        print( '        --recycle      Recycle all ports' )
+        print( '        --enable=<ports>       Enable <ports>' )
```

Comment:
Maybe add an example how you need ports to be transferred?
With spaces? commas?

Reply:
Missed it, maybe shift it up after the last argument that accept `ports`? your call